coala_quickstart/generation/Bears.py

```diff
@@ -7,7 +7,7 @@
 from coalib.output.printers.LogPrinter import LogPrinter
 
 
-def filter_relevant_bears(used_languages):
```

Comment:
see https://github.com/coala/coala/pull/3021 / https://github.com/coala/coala/issues/3012 for context of this change.